pkg/vetter/util/mtlspolicy/authPolicy_test.go

```diff
@@ -189,6 +190,21 @@ var _ = Describe("LoadAuthPolicies", func() {
 	})
 })
 
+var _ = Describe("GetClusterPortPolicies", func() {
+	It("should return port policies from the loaded AuthPolicies struct", func() {
+		loaded, err := LoadAuthPolicies([]*authv1alpha1.Policy{
+			apDefaultOn,
+			apFooOn,
+			apFooOff,
+			apFooBarOn,
+			apFooPortsBarOn,
+		})
+		Expect(err).To(Succeed())
+		portPolicies := loaded.GetClusterPortPolicies()
+		Expect(len(portPolicies)).To(Equal(1))
```

Comment:
`Expect(len(portPolicies)).To(Equal(1))` -> `Expect(portPolicies).To(HaveLen(1))`